tests/integration/operator-integ-suite-ocp.sh

```diff
@@ -24,14 +24,49 @@ set -u
 # Print commands
 set -x
 
-# deploy operator in OCP
-echo "--------------------------------"
-echo "Check that istio operator is running in OCP"
-echo "--------------------------------"
-"${WD}"/check-operator.sh
-
-# create a Istio Helm Install CP and test httpbin
-echo "--------------------------------"
-echo "Create a IstioHelmInstall Control Plane and test httpbin"
-echo "--------------------------------"
-"${WD}"/control-plane-test.sh
+NAMESPACE="${NAMESPACE:-istio-operator}"
+OPERATOR_NAME="${OPERATOR_NAME:-istio-operator}"
+CONTROL_PLANE_NS="${CONTROL_PLANE_NS:-istio-system}"
+
+BRANCH="${BRANCH:-maistra-3.0}"
```

Comment:
nit: unsed variable.

Hint: If you use vscode, install the `ShellCheck` extension. It will highlight these kind of "issues".